lib/Reference/CommentReferenceProvider.php

```diff
@@ -85,9 +85,14 @@ public function resolveReference(string $referenceText): ?IReference {
 			if ($ids !== null) {
 				[$boardId, $cardId, $commentId] = $ids;
 
-				$card = $this->cardService->find($cardId)->jsonSerialize();
-				$board = $this->boardService->find($boardId)->jsonSerialize();
-				$stack = $this->stackService->find((int) $card['stackId'])->jsonSerialize();
+				try {
+					$card = $this->cardService->find($cardId)->jsonSerialize();
+					$board = $this->boardService->find($boardId)->jsonSerialize();
+					$stack = $this->stackService->find((int) $card['stackId'])->jsonSerialize();
+				} catch (NoPermissionException $e) {
```

Comment:
NoPermissionException still needs to be imported

Reply:
Fixed